### ISSUE TITLE
Proper message when angular test.ts has invalid order instead of "TypeError: Cannot read property 'assertPresent' of undefined"

### ISSUE THREAD

**Issue by vovikdrg** [angular/zone.js#1147]
After I enabled ts-lint autofix with import sorting I got error in my tests "TypeError: Cannot read property 'assertPresent' of undefined"

// This file is required by karma.conf.js and loads recursively all the .spec and framework files

```
import { getTestBed } from '@angular/core/testing';
import { BrowserDynamicTestingModule, platformBrowserDynamicTesting } from '@angular/platform-browser-dynamic/testing';
import 'zone.js/dist/zone-testing';


declare const require: any;

getTestBed().initTestEnvironment(
    BrowserDynamicTestingModule,
    platformBrowserDynamicTesting()
);
const context = require.context('./', true, /\.spec\.ts$/);

context.keys().map(context);
```
After that i found https://github.com/angular/angular/issues/11317#issuecomment-419122951 which solved my issue but I am not one who have this problem

I would expect to see proper message that my "import 'zone.js/dist/zone-testing';" is wrong.

I also submitted feature-request to angular-cli but it was rejected. Such issues just make people to dislike angular while I want them to love it :)

**Comment by JiaLiPassion** [comment]
the reason is `tslint` just change the `import order`.
you need to `import 'zone.js/dist/zone-testing'` in the first line.

**Comment by vovikdrg** [comment]
@JiaLiPassion You are absolutely right, but my point is that proper error message should say that, how can I say what is wrong from this error message ?

**Comment by JiaLiPassion** [comment]
@vovikdrg, oh, you are right, I will change this message, thanks!
